terraso_backend/apps/core/models/landscapes.py

```diff
@@ -49,6 +51,9 @@ class Meta(SlugModel.Meta):
         }
 
     def save(self, *args, **kwargs):
+        if self.area_polygon:
+            self.area_scalar_m2 = calculate_geojson_feature_area(self.area_polygon)
```

Comment:
Question: @david-code Is this what you were talking about in the slack thread about dirty fields? It seems that right now every time the landscape data is updated the area_scalar field will be recalculated, right? so this will be triggered even if the area polygon value is the same, right?

Reply:
Yes, exactly. Looking a bit more at this, I think I will try installing the django-dirty-fields library. It seems like the best solution to me.